src/api/services/task_queue.py

```diff
@@ -34,7 +34,11 @@ def list_tasks_functions(self, context: Context, args) -> Tuple[list, MassEnergi
 
   def create_task(self, context, args) -> Tuple[dict, MassEnergizeAPIError]:
     if not args.get("job_name", None):
-      return None, MassEnergizeAPIError("Automatic proccess is required")
+      return None, MassEnergizeAPIError("Automatic process is required")
+    
+    if args.get("job_name") not in FUNCTIONS.keys():
+      return None, MassEnergizeAPIError("Automatic process is not valid")
```

Comment:
Please make this "Automatic task name not unique"

Reply:
line 39 is checking if the job_name supplied is in the `FUNCTIONS` dictionary so I think the error message on line 40 should read `Automatic task name not valid`. 🤷🏽